packages/dashboards/src/lib/common/pipes/dashboard-unit-conversion-pipe.ts

```diff
@@ -17,10 +17,14 @@ export class DashboardUnitConversionPipe implements PipeTransform {
      *
      * @returns The string representation of the converted value
      */
-    public transform = (value: string | number | undefined, units: UnitOption = "generic"): string => {
+    public transform = (
+        value: string | number | undefined,
+        units: UnitOption = "generic",
+        defaultThreshhold: number = DEFAULT_UNIT_CONVERSION_THRESHOLD
```

Comment:
you have typo in name of the argument (defaultThreshold) and you should use the enum we already have :UnitBase:

similar to this:
https://github.com/solarwinds/nova/blob/c82170c479d4579c0a2f4e0e94ce8b705823f973/packages/bits/src/services/unit-conversion.service.ts#L26